contracts/vcnote/VCNote.sol

```diff
@@ -168,6 +185,38 @@ contract VCNote is CErc20Delegate_VCNote {
         syncLendingLedger(dst);
     }
 
+    //////////////////////
+    ////////////// V2
+    function mintCNote(uint noteAmount) public {
+        require(admin == msg.sender, "VCNote::mintCNote: only admin");
+        _mintCNote(noteAmount);
+    }
```

Comment:
what is the purpose of this admin function?

Reply:
It's unlikely, but if someone accidentally sends a note to a cNote contract, the note becomes isolated and cannot be moved. 

In that case, the admin converts the note to a cnote and absorbs it into the pool.